js/trackers.js

```diff
@@ -179,13 +179,10 @@ function isRelatedEntity(parentCompany, currLocation) {
  * pull off any subdomains before comparison
  */
 function isFirstPartyRequest(currLocation, urlToCheck) {
-    let hostname1 = utils.extractHostFromURL(currLocation)
-    hostname1 = hostname1.split('.').slice(-2).join('.')
+    let currentLocationParsed = URLParser.parse(currLocation)
+    let urlToCheckParsed = URLParser.parse(urlToCheck)
 
-    let hostname2 = utils.extractHostFromURL(urlToCheck)
-    hostname2 = hostname2.split('.').slice(-2).join('.')
-
-    if (hostname1 === hostname2) {
+    if (currentLocationParsed.domain === urlToCheckParsed.domain) {
         return true
     }
 
```

Comment:
Some disambiguations: In trackers.js, there are a couple of references to `URLParser` within `isFirstPartyRequest()`. Am wanting to move towards eliminating/consolidating as many globals as possible for browserify switch, and since we're already using `utils` as a global, why don't we just wire up `URLParser.parse` in the `utils` exports? Otherwise other devs will come along and not know which to use, and we increase the # of globals.